### DISCUSSION TITLE
how do i build for wasm32-unknown-emscripten?

### DISCUSSION THREAD

**Discussion by JonnycatMeow** [emscripten-core/emscripten#20220] [General]
i am trying to build for vlc wasm and idk where to get it for mac please help

**Comment by sbc100** [comment]
Installation instructions from emscripten are here: https://emscripten.org/docs/getting_started/downloads.html.  The simplest way to install it is via emsdk.

As for how specifically to build vlc using emscripten, that would be much large question.  IIUC other folks have already done this in the past.  You might want to try asking on the vlc forums.
